### PR TITLE
Update messages of issue closers

### PR DESCRIPTION
**Pull request by michelleangela**

### CODE REVIEW

**Review by WardenGnaw — APPROVED** [review]

File: .github/workflows/by-design-closer-debugger .yml [hunk 1]
```diff
@@ -23,4 +23,4 @@ jobs:
           labels: by design,debugger
           ignoreLabels: language service,internal
           closeDays: 0
-          closeComment: "This issue has been closed automatically because it's labeled as 'by design'."
+          closeComment: "This issue has been closed because the described behavior was determined as by design."
```
Comment (bobbrow) [inline .github/workflows/by-design-closer-debugger .yml]: "was determined to be by design"

File: .github/workflows/by-design-closer.yml [hunk 2]
```diff
@@ -23,6 +23,6 @@ jobs:
           labels: by design
           ignoreLabels: debugger,Feature Request,more info needed,internal
           closeDays: 60
-          closeComment: "This issue has been closed automatically because it's labeled as 'by design' and has not had recent activity."
+          closeComment: "This issue has been closed because the described behavior was determined as by design."
```
Comment (bobbrow) [inline .github/workflows/by-design-closer.yml]: "was determined to be by design"

File: .github/workflows/duplicate-closer.yml [hunk 3]
```diff
@@ -23,6 +23,6 @@ jobs:
           labels: duplicate
           ignoreLabels: debugger,Feature Request,more info needed,by design,internal
           closeDays: 60
-          closeComment: "This issue has been closed automatically because it's labeled as a 'duplicate' and has not had recent activity."
+          closeComment: "This issue has been closed because it is a duplicate of another one."
```
Comment (bobbrow) [inline .github/workflows/duplicate-closer.yml]: "of another issue we are tracking"

File: .github/workflows/question-closer-debugger.yml [hunk 4]
```diff
@@ -24,6 +24,6 @@ jobs:
           ignoreLabels: language service,internal
           involves: wardengnaw,pieandcakes,calgagi
           closeDays: 14
-          closeComment: "This issue has been closed automatically because it's labeled as a 'question' and has not had recent activity."
+          closeComment: "For housekeeping purposes, this issue has been closed because it is a question and has not had recent activity."
```
Comment (bobbrow) [inline .github/workflows/question-closer-debugger.yml]: We don't mention "housekeeping purposes" on any of the other messages, so maybe we don't need it here either.  "This issue has been closed..." is ok, I think.

**Pull request merged**